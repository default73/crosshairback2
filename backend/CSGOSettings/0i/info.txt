Имя~ЧжиВэй Лю (ZhiWei Liu)
DPI:~400
Частота мыши:~1000Hz
Чувствительность мыши в игре:~2.27
Чувствительность мыши в зуме:~1.2
Чувствительность мыши в Windows:~6/11
Ускорение мыши:~0
m_rawinput:~1
Разрешение:~1280 x 960
Соотношение сторон:~4:3
Формат изображения:~Растянутое
Частота обновления:~240Hz
sharecode~CSGO-HH998-jQePF-udosk-tJ585-tXcNE
st~https://steamcommunity.com/profiles/76561198031156212
Команда~Без команды
Страна~Китай
