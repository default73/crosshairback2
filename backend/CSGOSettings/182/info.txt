Имя~Игнасио Риус Вердуна (Ignacio Rius Verduna)
DPI:~800
Частота мыши:~1000Hz
Чувствительность мыши в игре:~1.5
Чувствительность мыши в зуме:~1.2
Чувствительность мыши в Windows:~6/11
Ускорение мыши:~0
m_rawinput:~1
Разрешение:~1280 x 960
Соотношение сторон:~4:3
Формат изображения:~Растянутое
Частота обновления:~240Hz
sharecode~CSGO-y7DLq-a7ZiG-GN5Wc-8F8Qy-VtjjD
st~https://steamcommunity.com/profiles/76561198840163201
twitch~https://www.twitch.tv/182cs
tw~https://twitter.com/182CSGO
in~https://instagram.com/ignacioriusv
Команда~Без команды
Страна~Аргентина
