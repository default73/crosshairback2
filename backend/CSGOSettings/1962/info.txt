Имя~Габриэль Синополи (Gabriel Sinopoli)
DPI:~800
Частота мыши:~1000Hz
Чувствительность мыши в игре:~1.55
Чувствительность мыши в зуме:~1
Чувствительность мыши в Windows:~6/11
Ускорение мыши:~0
m_rawinput:~1
Разрешение:~1280 x 960
Соотношение сторон:~4:3
Формат изображения:~Черные полосы
Частота обновления:~240Hz
sharecode~CSGO-cSh45-xzmVq-sWqEH-iCOMR-udBAA
st~https://steamcommunity.com/profiles/76561198281767680
twitch~https://www.twitch.tv/1962cs
fb~https://www.facebook.com/1962twm
tw~https://twitter.com/1962cs
in~https://www.instagram.com/1962cs/
Команда~Без команды
Страна~Аргентина
