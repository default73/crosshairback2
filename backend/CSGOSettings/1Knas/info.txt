Имя~Герман Хаттевиг (Herman Hattevig)
DPI:~800
Частота мыши:~1000Hz
Чувствительность мыши в игре:~1.3
Чувствительность мыши в зуме:~0.7
Чувствительность мыши в Windows:~6/11
Ускорение мыши:~0
m_rawinput:~1
Разрешение:~1280 x 960
Соотношение сторон:~4:3
Формат изображения:~Растянутое
Частота обновления:~240Hz
sharecode~CSGO-TYM8H-dChwz-6Wt9W-22kdk-pL3ED
Параметры запуска~+fps_max 0 -allow_third_party_software
st~https://steamcommunity.com/profiles/76561198165967286
tw~https://twitter.com/1KnasCSGO
Команда~7AM
Страна~Швеция
