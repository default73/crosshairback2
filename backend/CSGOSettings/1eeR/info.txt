Имя~Александр Нагорный ( Alexander Nagorny)
DPI:~400
Частота мыши:~1000Hz
Чувствительность мыши в игре:~2.3
Чувствительность мыши в зуме:~0.9
Чувствительность мыши в Windows:~6/11
Ускорение мыши:~0
m_rawinput:~1
Разрешение:~1280 x 960
Соотношение сторон:~4:3
Формат изображения:~Растянутое
Частота обновления:~240Hz
sharecode~CSGO-9AeJ7-YB4r9-NaOjQ-8UNiY-cn3cQ
Параметры запуска~-freq 240 -novid -tickrate 128 -allow_third_party_software
st~https://steamcommunity.com/profiles/76561198837117408
vk~https://vk.com/1eeer24
twitch~https://www.twitch.tv/1eer24
yt~https://www.youtube.com/channel/UCGeIRw5f-QzBGQejlhMGoBQ
in~https://www.instagram.com/1eer24
Команда~Nemiga Gaming
Страна~Беларусь
