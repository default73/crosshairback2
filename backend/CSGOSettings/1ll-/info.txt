Имя~Илья Воронов (Ilya Voronov)
DPI:~400
Частота мыши:~1000Hz
Чувствительность мыши в игре:~2.2
Чувствительность мыши в зуме:~1
Чувствительность мыши в Windows:~6/11
Ускорение мыши:~0
m_rawinput:~1
Разрешение:~1280 x 960
Соотношение сторон:~4:3
Формат изображения:~Растянутое
Частота обновления:~240Hz
sharecode~CSGO-N75N3-kVnLt-ZVRhG-JeT7a-tUfOQ
Параметры запуска~-freq 240 -novid -console -tickrate 128 +clientport 27027 +fps_max 0
st~https://steamcommunity.com/profiles/76561198799643789
vk~https://vk.com/id233143688
twitch~https://www.twitch.tv/1llusha_cs
tw~https://twitter.com/1l_CSGO
in~https://www.instagram.com/il_voronoff/
Команда~Без команды
Страна~Россия
