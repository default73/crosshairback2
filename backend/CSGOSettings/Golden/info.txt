Имя~Майкил Селим (Maikil Selim)
DPI:~400
Частота мыши:~1000Hz
Чувствительность мыши в игре:~2
Чувствительность мыши в зуме:~1
Чувствительность мыши в Windows:~6/11
Ускорение мыши:~0
m_rawinput:~1
Разрешение:~1024 x 768
Соотношение сторон:~4:3
Формат изображения:~Черные полосы
Частота обновления:~240Hz
sharecode~CSGO-RvGde-NQ4ev-xPaKA-xr9SR-5EWzE
st~https://steamcommunity.com/id/GOLDNCSGO/
twitch~https://www.twitch.tv/goldenmajk
tw~https://twitter.com/goldenmajk
in~https://www.instagram.com/goldenmajk/
Команда~VELOX Argentina
Страна~Швеция
